stancon_2020/InferenceData_Stancon2020_cat.ipynb

```diff
@@ -12,33 +12,30 @@
     "\n",
```

Comment:
<p>So I cant exactly read catalan in the slide above but I surmise this says, for legibility fix the load. I would say don't worry about it now since we already recorded the other ones as is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_misc/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>